docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+# WebView
+
+You can access web pages and web content in your application. You can also use various features for web browsing, such as loading and displaying web pages and navigating through the browsing history.
+
+The main features of WebView are as follows:
+
+- Initializing and Creating WebView
+
+    You can [initialize a WebView](#initialize) and [create a WebView object](#initialize).
+
+- Managing Web Page Loading
+
+    You can [load a web page](#load) with requested URL and [register event handlers](#load) to load events.
```

Comment:
@younghajung 
For consistency sake, revert to "loading events", change to:

You can [load a web page](#load) with requested URL and [register event handlers](#load) for loading events.

Reply:
Done.

Reply:
@younghajung

Thank you for fixing the comments. I have one request, going forward please do not force push a commit that deletes the history, instead push it as a new commit.

Reply:
OK, I get it.